sycl/doc/extensions/experimental/sycl_ext_oneapi_properties.asciidoc

```diff
@@ -200,11 +200,10 @@ inline constexpr boo_key::value_t<Ts...> boo;
 === Property traits
 
 All runtime and compile-time-constant properties must have a specialization of 
-`is_property_key` and `is_property_value` that inherits from
-`std::true_type`, and they must have a specialization of `is_property_key_of`
-and `is_property_value_of`
-that inherits from `std::true_type` for each SYCL runtime class that the
-property can be applied to. All have a base case which inherits from `std::false_type`.
+`is_property_key` that inherits from `std::true_type`, and they must have a
+specialization of `is_property_key_of` that inherits from `std::true_type` for
+each SYCL runtime class that the property can be applied to. All have a base
+case which inherits from `std::false_type`.
```

Comment:
I don't think it would be wise. The remaining traits (after this PR) still apply to the keys, while future `is_property` would reference the property/value and the keys will be hidden as implementation details.